routes/user.go

```diff
@@ -1078,6 +1092,16 @@ func (fes *APIServer) GetSingleProfile(ww http.ResponseWriter, req *http.Request
 		res.IsGraylisted = true
 	}
 
+	var userMetadata *UserMetadata
+	userMetadata, err = fes.getUserMetadataFromGlobalState(publicKeyBase58Check)
+	if err != nil {
+		_AddBadRequestError(ww, fmt.Sprintf("AdminUpdateTutorialCreator: error getting usermetadata for public key: %v", err))
+		return
+	}
```

Comment:
We probably don't want to do this on every call. It's only relevant for admins. Additionally, getting user metadata from global state + decoding will make this request take longer + put more strain on the infrastructure.

Let's only get this if the reader is an admin? This may be an over optimization but may as well keep it tight for now.